integration-test/src/test/java/com/datastrato/gravitino/integration/test/web/ui/Pages/MetalakePage.java

```diff
@@ -265,6 +279,11 @@ public boolean verifyLinkToCatalogsPage(String name) {
       return nameLink.isDisplayed() && isUrl;
     } catch (Exception e) {
       return false;
+    } finally {
+      // Back to homepage
+      String xpath = "//button[@data-refer='back-home-btn']";
+      WebElement backHomeBtn = driver.findElement(By.xpath(xpath));
```

Comment:
Whether need define `backHomeBtn` to be a MetalakePage Class vaiable?

Reply:
I added the backHomeBtn Class in CatalogsPage.java, I will modify when the PR is submited.